src/ingest_validation_tools/table-schemas/sample.yaml

```diff
@@ -2,283 +2,111 @@ doc_url: TODO
 fields:
 -
   heading: IDs
-  name: donor_id
-  description: TODO
-  constraints:
-    required: True
-    pattern: '[A-Z]+[0-9]+'
--
   name: sample_id
   description: TODO
   constraints:
-    required: True 
+    required: True
     pattern: '([A-Z]+[0-9]+)-(BL|BR|LB|RB|HT|LK|RK|LI|LV|LL|RL|LY\d\d|SI|SP|TH|TR|UR|OT)(-\d+)+(_\d+)?' 
 -
   heading: Donor
-  name: donor_source_institution
-  constraints:
-    required: True
-    enum: 
-      - A # Washu OPO
-      - B # Rochester OPO
-      - C # Vanderbilt   
-  description: Name of institution that sourced the donor (anonymized) - TODO - This could be trivially de-anonymized. Is this for this release, or something in the future?
-
-  TODO_summation: yes  
-  TODO_nils: Should this really be part of the submission if the mapping is that simple?  
-  TODO_type: enum  
-  TODO_shin: this grid may be used for future project -- that's what Ajay said  
-  TODO_sanjay:
--
-  name: donor_vital_state 
+  name: vital_state 
   constraints:
     required: True 
     enum:
       - living
       - deceased  
-  description: identify the vital state of the donor  
-
-  TODO_summation: yes  
-  TODO_nils:   
-  TODO_type: enumeration of allowed values  
-  TODO_shin:   
-  TODO_sanjay:
+  description: Identify the vital state of the donor.  
 -
-  name: donor_health_status 
+  name: health_status 
   constraints:
     required: True 
     enum:
       - cancer
       - relatively healthy
       - chronic illness  
-  description: Patient's physical condition immediately preceding death. TODO - Confirm that you want precisely these three mutually exclusive values? I thought HuBMAP included only healthy tissue?
-  
-  TODO_summation: yes  
-  TODO_nils:   
-  TODO_type: enumeration of allowed values  
-  TODO_shin:   
-  TODO_sanjay:
+  description: Patient's physical condition immediately preceding death.
+-
+  name: body_imaging 
```

Comment:
I thought we had agreed to drop this field but I am not 100% sure.

Reply:
I wondered too, just didn't have it explicit in notes; will drop.